.gitpod.yml

```diff
@@ -7,10 +7,12 @@ ports:
 - port: 5900
   onOpen: ignore
 tasks:
-- init: >
-    yarn
-  command: >
+- command: >
     jwm &
+    yarn clean && 
+    git submodule update --remote &&
```

Comment:
This way by default we always use the very latest vscode master. I figured it was not much use to pre-build everything in the init block just to do "git clean" and rebuild/package based on latest each time a Gitpod container for this repo is deployed (vscode master is a fast-moving target) . WDYT?

Reply:
Could this be done in `init`?

Reply:
done